### PR TITLE
Files implementation

### PR DESCRIPTION
**Pull request by Andrewangeta**

### CODE REVIEW

**Comment by Andrewangeta** [comment]
This addresses https://github.com/vapor-community/stripe-provider/issues/66

File: Sources/Stripe/API/Routes/FileRoutes.swift [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        var body: Data = Data()
+        
+        guard let fileType = MediaType.fileExtension(type.rawValue) else {
+            throw StripeUploadError.unsupportedFileType
+        }
+        
+        // Fordm data structure found here.
+        // https://www.w3.org/TR/html401/interact/forms.html#h-17.13.4
+        
+        let boundary = "Stripe-Vapor-\(UUID().uuidString)"
```
Comment (anthonycastelli) [inline Sources/Stripe/API/Routes/FileRoutes.swift:66]: Vapor has a Mutlipart library that you can import to take care of all this heavy lifting. I'm not sure how it all works in Vapor 3, but I know it's still around.

**Review by anthonycastelli — APPROVED** [review]

File: Sources/Stripe/API/Routes/FileRoutes.swift (hunk 1 above)
Comment (Andrewangeta) [reply Sources/Stripe/API/Routes/FileRoutes.swift:66]: Ah I see. I just checked it out and I'd have to have multiple structs to be able to take advantage of  it. I'd rather just hand write it. I've done this on iOS a lot too.

Comment (anthonycastelli) [reply Sources/Stripe/API/Routes/FileRoutes.swift:66]: Sounds good to me. If it becomes problematic we can look into swapping it over.

**Pull request merged**